blocks/collection-hero/collection-hero.css

```diff
@@ -0,0 +1,72 @@
+.collection-hero {
+    position: relative;
+}
+
+@media screen and (min-width: 1200px) {
+    .collection-hero {
+        height: 100vh;
+    }
+}
+
+.collection-hero img {
+    width: 100%;
+    height: 260px;
+    object-fit: cover;
+    filter: brightness(70%);
+}
+
+@media screen and (min-width: 600px) {
+    .collection-hero img {
+        height: 100%;
+    }
+}
+
+.section.collection-hero-container {
+    max-width: unset;
+    padding: unset;
+}
+
+.section > div.collection-hero-wrapper {
+    max-width: unset;
+}
+
+.collection-hero .hero-collection-overlay {
+  position: absolute;
+  top: 20%;
+  left: 5%;
+  color: #fff;
+  font-size:40px;
+  font-weight: 400;
+  margin-bottom: 8px;
+}
+
+  .collection-hero h1.hero-collection-title{
+    font-size: 26px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+  }
+
+  .collection-hero p.hero-collection-subtitle{
+    font-size: 16px;
+    letter-spacing: 1px;
+    line-height: 28px;
+  }
+
+@media (min-width: 1200px){
```

Comment:
There are two media queries for 1200px. One is specifically for screens (i.e. not applied when printing) and the other applies regardless of the device. Is this intentional?

Reply:
Not intentional.  I will fix it by adding `@media screen and `